api/data/seattle_db.go

```diff
@@ -2,53 +2,78 @@ package data
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	"github.com/gobuffalo/nulls"
 	"github.com/jackc/pgx/v4"
 )
 
 // SeattleOfficer is the object model for SPD officers
 type SeattleOfficer struct {
+	Date			time.Time    `json:"date,omitempty"`
 	BadgeNumber     string       `json:"badge_number,omitempty"`
-	FirstName       string       `json:"first_name,omitempty"`
-	MiddleName      nulls.String `json:"middle_name,omitempty"`
-	LastName        string       `json:"last_name,omitempty"`
+	FullName		string       `json:"full_name,omitempty"`
 	Title           string       `json:"title,omitempty"`
 	Unit            string       `json:"unit,omitempty"`
 	UnitDescription nulls.String `json:"unit_description,omitempty"`
+	FirstName       string       `json:"first_name,omitempty"`
+	MiddleName      nulls.String `json:"middle_name,omitempty"`
+	LastName        string       `json:"last_name,omitempty"`
 }
 
 // SeattleOfficerMetadata retrieves metadata describing the SeattleOfficer struct
-func (c *Client) SeattleOfficerMetadata() []map[string]string {
-	return []map[string]string{
-		{
-			"FieldName": "badge_number",
-			"Label":     "Badge Number",
-		},
-		{
-			"FieldName": "first_name",
-			"Label":     "First Name",
-		},
-		{
-			"FieldName": "middle_name",
-			"Label":     "Middle Name",
-		},
-		{
-			"FieldName": "last_name",
-			"Label":     "Last Name",
-		},
-		{
-			"FieldName": "title",
-			"Label":     "Title",
-		},
-		{
-			"FieldName": "unit",
-			"Label":     "Unit",
-		},
-		{
-			"FieldName": "unit_description",
-			"Label":     "Unit Description",
+func (c *Client) SeattleOfficerMetadata() DepartmentMetadata {
+	var date time.Time
+	err := c.pool.QueryRow(context.Background(),
+		`
+			SELECT date
+			FROM seattle_officers
+			WHERE date = (select max(date) from seattle_officers);
```

Comment:
I might be wrong, but I think this can just be 
```suggestion
			SELECT max(date) as date
			FROM seattle_officers
```

No need to subquery if we don't have to!

Reply:
It appears you are indeed correct. I am bad at SQL haha, thanks for catching this.

Reply:
I'm not a great SQL alchemist but I can get by haha